packages/sanity/src/core/studio/StudioErrorBoundary.tsx

```diff
@@ -19,6 +19,17 @@ import {CorsOriginError} from '../store'
 import {isRecord} from '../util'
 import {CorsOriginErrorScreen, SchemaErrorsScreen} from './screens'
 
+/**
```

Comment:
@juice49 added this comment to clarify that although on the face it looks like this might not be available, in practice it always will be when it's needed